cpp/include/cudf/null_mask.hpp

```diff
@@ -95,7 +95,7 @@ rmm::device_buffer create_null_mask(
 
 /**
  * @brief Sets a pre-allocated bitmask buffer to a given state in the range
- *  `[begin_bit, end_bit)`
+ * `[begin_bit, end_bit)`.
```

Comment:
Technically we don't expect `@brief` statements to end in a period since they are considered titles and not descriptions.
(Looks like I need to update the doc guide on this)

Reply:
Thanks for letting me know. I had added this for consistency across other docstrings in the file.